solutions/standard/variables.tf

```diff
@@ -8,35 +8,40 @@ variable "ibmcloud_api_key" {
   sensitive   = true
 }
 
+variable "existing_resource_group_name" {
+  type        = string
+  description = "The name of an existing resource group to provision resource in."
+  default     = "Default"
+  nullable    = false
+}
+
 variable "prefix" {
   type        = string
-  description = "The prefix to add to all resources that this solution creates. To not use any prefix value, you can set this value to `null` or an empty string."
-  default     = "icr"
+  description = "The prefix to add to all resources that this solution creates (e.g `prod`, `test`, `dev`). To not use any prefix value, you can set this value to `null` or an empty string."
```

Comment:
Description should include info about the required regex. See the code snippet in https://terraform-ibm-modules.github.io/documentation/#/da-implementation-guidelines?id=prefix-in-deployable-architecture